recipes/quazip/all/conanfile.py

```diff
@@ -0,0 +1,73 @@
+from conans import ConanFile, CMake, tools
+import os
+
+class QuaZIPConan(ConanFile):
+    name = "quazip"
+    description = "A simple C++ wrapper over Gilles Vollant's ZIP/UNZIP package\
+                    that can be used to access ZIP archives."
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/stachenov/quazip"
+    license = "LGPL-2.1-STATIC_LINKING_EXCEPTION"
```

Comment:
```suggestion
    license = "LGPL-2.1-linking-exception"
```

There is a official short name for it: https://spdx.org/licenses/LGPL-3.0-linking-exception.html